src/graph/graph.h

```diff
@@ -61,7 +61,7 @@ class Graph
 
     struct ScheduleComparison
     {
-        bool operator()(const pair<uint64_t, int64_t>& lhs, const pair<uint64_t, int64_t>& rhs)
+        bool operator()(const pair<uint64_t, int64_t>& lhs, const pair<uint64_t, int64_t>& rhs) const
```

Comment:
Thanks for the fix.

Just a nit to not have a too long line:
```cpp
        bool operator()(const pair<uint64_t, int64_t>& lhs,
                        const pair<uint64_t, int64_t>& rhs) const
```

Also, would you apply the same fix to line 79? i.e.:
```cpp
        bool operator()(const pair<Vertex, int64_t>& lhs,
                        const pair<Vertex, int64_t>& rhs) const
```

Reply:
I haven't encountered that problem yet; if I do, I will make the necessary changes.